### PR TITLE
HAST-175: NuGet publishing fixes in Lombiq.GitHub.Actions

### PR DESCRIPTION
**Pull request by Piedone**
[HAST-175](https://lombiq.atlassian.net/browse/HAST-175)

[HAST-175]: https://lombiq.atlassian.net/browse/HAST-175?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/actions/spelling/excludes.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Lombiq\.VueJs\.Tests\.UI/Assets/Media/.*\.mjpeg$
+^\Qsrc/Modules/Lombiq.Hosting.Tenants/Readme.md\E$
```
Comment (sarahelsaig) [inline .github/actions/spelling/excludes.txt:3]: Why is the `\Q...\E` necessary here? imo even leaving the dots unescaped should be fine, it's not like we expect any realistic false positives. But at any rate for consistency's sake, let's either wrap every line like here, or none of them.

**Review by sarahelsaig — CHANGES_REQUESTED** [review]

File: .github/actions/spelling/excludes.txt (hunk 1 above)
Comment (Piedone) [reply .github/actions/spelling/excludes.txt:3]: Yeah, it's probably an overkill though dots are escaped in most of the other lines. I'll wrap them.

Comment (0liver) [reply .github/actions/spelling/excludes.txt:3]: I would choose simplicity over such uncommon configuration.

What's the real problem that using `\Q...\E` solves?

Comment (0liver) [reply .github/actions/spelling/excludes.txt:3]: Btw, where's the documentation on `\Q...\E`? The only mention I've found was here: https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples%3A-excludes, and it doesn't explain what it does there.

Comment (sarahelsaig) [reply .github/actions/spelling/excludes.txt:3]: It's a feature in _some_ regex engines, though it doesn't work in .NET regex expressions.
https://www.regular-expressions.info/characters.html#:~:text=Some%20flavors%20also%20support%20the

Comment (0liver) [reply .github/actions/spelling/excludes.txt:3]: I didn't know that one, thanks!

**Comment by sarahelsaig** [comment]
merge conflict here, otherwise approved

**Pull request merged**